index.js

```diff
@@ -0,0 +1,20 @@
+const { absolutePath, archiveTrue, getLinks } = require('./src/mdlink.js');
+const { validate } = require('./src/options.js');
+
+const mdLinks = (path, options) => new Promise((resolve) => {
+  const absolute = absolutePath(path);
+ //console.log(absolute); 
+  const checkpath = archiveTrue(absolute);
```

Comment:
Si la variable muta( cabia de valor ) definela con let
```suggestion
  let checkpath = archiveTrue(absolute);
```